rtl/eclrtl/eclrtl.cpp

```diff
@@ -558,8 +558,8 @@ double rtlRoundTo(const double x, int places)
 {
     if (x < 0)
         return -rtlRoundTo(-x, places);
-    volatile double tt = x * kk;
-    double x0 = x + tt;
+//    volatile double tt = x * kk;
```

Comment:
I suspect the following code should be using x0 instead of x, but not sure if we should change it, or remove these comments?